Introduction (#56)
WhitakerLab/whitakerlab.github.io: _posts/blog/2019-06-20-Hello!I am Yini He.md

---
layout: post

> title: Hello^0^I am Yini He!

Reviewer: I don't quite follow what "^0^" means? If you want it in can you put spaces around it?

Author: It is a smile icon which same with :) .

categories: blog
excerpt: An introduction to the Whitaker lab's newest member
image:
  feature:
link:
date: 2019-06-20
modified:
share: true
author: Yini-He
---

## Who am I

> --I am who I am（我就是我）

Reviewer: This is a quote, right? I think it would render really nicely as a quote:

```suggestion
> I am who I am（我就是我）
```

> My name is Yini He（何旖旎）.I’m a PhD student with a major in neuroimaging at the University of Electronic Science and Technology of China (UESTC).It is truly amazing for me to study in the WhitakerLab at the Alan Turing Institute as a visiting student,jointly funded by Chinese Scholarship Council (CSC) and the Newton Fund.I am enthusiastic about  the relationship between individual traits, the brain and emotions, and particularly the associated risks of developing mental disorders.

Reviewer: Hi @minnieho1115 - please can you put all the sentences on new lines?

Also, watch for double spaces instead of single spaces, and make sure to put a space after a `.` or `,`.

```suggestion
My name is Yini He (何旖旎).
I’m a PhD student with a major in neuroimaging at the University of Electronic Science and Technology of China (UESTC).
It is truly amazing for me to study in the WhitakerLab at the Alan Turing Institute as a visiting student, jointly funded by Chinese Scholarship Council (CSC) and the Newton Fund.
I am enthusiastic about the relationship between individual traits, the brain and emotions, and particularly the associated risks of developing mental disorders.
```

Author: Thank Kirstie!

> ## Where I come here  

Reviewer: Where I come here doesn't make grammatical sense. Do you mean "Where I come from"?

> --As different as night and day（冰火两重天）

Reviewer: Quote as above?

I was born in Changsha city of China which famous for the spicy dishes and excessively distinct seasons. And then I moved to Haikou, which famous for the gentlest sea wind and spring-like seasons. As its title indicates, these two cities have extreme difference, just like a song plays from a gentle and poetic melody to passionate and fiery bursts, fully influencing the two sides of my personality. I am happy and passionate at work, at the same time, enjoy a totally peaceful and quiet life. 

<figure>

>   <img src="/images/YiniHe/1coconut4.jpg">

Reviewer: Can you update the image to include the link to image on the internet? You can also use "<figcaption>" to caption the image, it doesn't need to be in the picture itself.

</figure>

> ## What's my hobby

Reviewer: ```suggestion
## What are my hobbies?
```

> -- The more National, the more International.（民族的就是世界的）

Reviewer: Quote as above.

I love travelling around the world. The coexistence of the classics with the prosperity of London attracts me incredibly.  The people there are gentle and full of artistic temperament.  Modern and beautiful Vancouver attracts me just as much.  The people there understand the beauty of cultural integration.  Da Nang beckons to me with it’s mysteriousness and simplicity. The people there are warm and kind. 

<figure>

>   <img src="/images/YiniHe/2panda4.jpg">

Reviewer: Super cute! 🐼

</figure>
                  
At the same time, I am interested in all kinds of traditional Chinese culture, with its food, traditional clothing, calligaraphy, tea and Guqin(an ancient instrument of China). With a history of 5000 years, China has preserved its ancient philosophy and beautiful civilization.  More often, I am fascinated by the merging of Chinese and western cultures. I used to play the ukelele wearing Qipao(a traditional dress of China).  I also tried cooking a Chinese meal with fish and chips. 

<figure>
  <img src="/images/YiniHe/3qipao3.jpg">
</figure>

> # Life and work goals

Reviewer: Second level heading instead of 1st.
```suggestion
## Life and work goals
```

> --Ideal comes from life（理想源于生活）

Reviewer: Quote as above.

> My university major is psychology.  In my senior year, I decided to go to a special school and work as an intern teacher there.  The students at this school were suffering from mental disorder. This was the first time that I truly understood the importance of mental health at heart.  I love those children even when some of them say things that are illogical and incoherent, while still others just always sat in a corner. But there are those who have great artistic talents.  Some of them are attractive and have a friendly disposition, willing to be best friends with me. Ever since then, I’ve always thought about these teenagers and hope that they could have a better life with the help of medical and therapeutic advancements for mental health. 

Reviewer: I've put these sentences on new lines and adjusted the language around "special school" to make it a little more inclusive. (It isn't generally accepted to say that people with mental health conditions or learning difficulties are "suffering"). BUT please check to make sure that I've not changed the meaning 😄 

```suggestion
My university major is psychology.
In my senior year, I decided to go to a special school and work as an intern teacher there.
The students at this school all had learning difficulties and some other mental health conditions.
This was the first time that I truly understood the importance of mental health at heart.
I love those children for all their unique differences.
There were some who had great artistic talents.
Others would sit quietly by themselves.
Some of the kids had a really friendly disposition and wanted to be best friends with me.
Ever since then, I’ve always thought about these teenagers and hope that they could have a better life with the help of social, medical and therapeutic advancements for mental health. 
```

<figure>
  <img src="/images/YiniHe/4yuanmingyuan3.jpg">
</figure>

# My research experience

> Look at the fun of work, so we have fun every day(快乐生活，快乐工作)

Reviewer: ```suggestion
Look at the fun of work, so we have fun every day (快乐生活，快乐工作)
```

Ever since my experience as an intern at the school for special children, I have been concerned about the susceptibility to mental illness.  My major was modified to Neuroimaging.  The brilliance of how the brain works totally amazed me. I became very interested in regions of the brain, the connectivity and the fiber bundle of the brain where the source for the link on human behavior and emotion could be found and studied.

I really appreciate that I have a great supervisor who is Tianzi Jiang.  He is always rigorous and enthusiastic about Neuroscience. I have to say from the heart, that he is one of the best role model of mine.  

> Now I came to the Turing Institute and I was pleasantly surprised here. I love Kirstie’s energy and she always give supports to me. She told me that I should do what I am most interested in. I hope to learn from her how to do the most convincing research. I also hope to learn from her passion to try to change the world.

Reviewer: New paragraph?

Author: Yes.It is a new paragraph.Thank you very much for your careful revision.I've learned a lot from your advice
